just-install.json

```
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
```

foobar2000 uses NSIS for it's installer.

```suggestion
        "kind": "nsis",
```